src/rpc/RPCHelpers.hpp

```diff
@@ -747,4 +749,21 @@ getDeliveredAmount(
     uint32_t date
 );
 
+/**
+ * @brief Get Array of Credential objects
+ *
+ * @param credID Array of CredentialID's to parse
+ * @param backend backend interface
+ * @param info The ledger header
+ * @param yield The coroutine context
+ * @return Array of credential objects, error if failed otherwise
+ */
+std::expected<ripple::STArray, Status>
+fetchCredentialArray(
```

Comment:
Is there any specific reason why this is not in CredentialHelpers? 

I'd love to try and avoid adding more things to RPCHelpers (or any *Helpers). Imo adding a new file like you did is a better choice than adding to `RPCHelpers` that are already out of control in size and contain multiple functions that are only used once (or never) :rage4: 

Also, if the functions are to only be used in one place, maybe better keep them in the `impl` of the RPC handler (or other unit) that uses the function? Is there a real reason to create more shared helpers here?
I'm not opposed the current design (and we can always change it later) but in an ideal world i'd like to evict as many functions from `*Helpers` as possible and give them a new home in `impl` of their relevant handlers/feeds 👍

Reply:
I think I left it in RPC helpers because it uses all the backend interface/ledger/context yield stuff, but yea, ill move it to credentialHelpers 😄